recipe/meta.yaml

```diff
@@ -14,10 +14,11 @@ source:
     sha256: 189f3b98ed6b15290d4fd4dbab687c87de268b5c9f2f1c93a7c02710da062785
 
 build:
-  number: 0
+  number: 1
   script:
     - pushd {{ pkg_src }}
-    - go install -v -ldflags="-w -s -X main.version=conda-forge" ./cmd/task
+    - go install -v -ldflags="-w -s -X main.version=conda-forge" ./cmd/task  # [win]
+    - go build -v -ldflags="-w -s -X main.version=conda-forge" -o $PREFIX/bin/task ./cmd/task  # [not win]
```

Comment:
Not a go expert here. Do we not need to call `go install on Unix?

Reply:
`go install` refuses to install binaries for foreign architectures, so we install the binaries manually. Under `win` we can simply install instead.